src/main/java/com/hw/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -0,0 +1,13 @@
+package com.hw.bookstore.dto;
+
+import java.math.BigDecimal;
+
+public record CreateBookRequestDto(
+        String title,
+        String author,
+        String isbn,
+        BigDecimal price,
+        String description,
+        String coverImage

```

Comment:
here you should validate RequestDto (instead entity validation). Don't forget about annotation Valid in the corresponding controller

Reply:
I'll add validation in the next PRs.